tests/integration_frameworks/test_openai.py

```diff
@@ -1,15 +1,45 @@
-from utils import context, missing_feature, scenarios, features
+from utils import scenarios, features
 
 import pytest
+import os
 
 from utils.docker_fixtures import FrameworkTestClientApi, TestAgentAPI
 
 
+@pytest.fixture
+def library_env() -> dict[str, str]:
+    """Pass on the real API key for cassette generation for new tests."""
+
+    return {
+        "OPENAI_API_KEY": os.getenv("OPENAI_API_KEY", "<not-a-real-key>"),
```

Comment:
QQ : as the environment is handled by `IntegrationFrameworksScenario`, is this value still needed ?